install

```diff
@@ -125,8 +115,7 @@ do_install()
   rosinstall_generator ${ROS_CONFIGURATION} ${ROS_EXTRA_PACKAGES} --rosdistro ${ROS_DISTRO} --deps --tar > ${WS}.rosinstall
   wstool init -j8 src ${WS}.rosinstall
 
-  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+. Although formulae and patches
-  # exist to build Ogre 1.9, rviz, and Gazebo together, I have found this approach to be the most solid and consistent.
+  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+.
   if [ -d src/rviz ]; then
     pushd src/rviz/ogre_media/models
     curl https://raw.githubusercontent.com/ros-visualization/rviz/hydro-devel/ogre_media/models/rviz_cone.mesh > rviz_cone.mesh
```

Comment:
Sorry for the delayed response; I missed this at the time.

Most of our gazebo testing at the moment is on trusty with ogre 1.8, but we have also made lots of fixes so that it works with ogre 1.9, which is the default on xenial. So gazebo is compatible with ogre 1.8-1.9 on linux; I think the main issue is testing the homebrew formulae for ogre 1.8/1.9 in https://github.com/osrf/homebrew-simulation/pull/24 which hasn't been a huge priority for me, since the 1.7.4 formula still seems to work for gazebo. I'm happy to help review that pull request, but I probably won't take the lead any time soon.